scripts/enrich_match_data.py

```diff
@@ -0,0 +1,242 @@
+import os
+import sys
+import tqdm
+import json
+import pickle
+import datetime
+import jsonlines
+import wikipedia
+from wikipedia.exceptions import DisambiguationError, PageError
+from typing import List, Dict
+
+from nltk.corpus import stopwords
+from nltk.tokenize import RegexpTokenizer
+
+from emma.OntoEmma import OntoEmma
+import emma.utils.string_utils as string_utils
+from emma.paths import StandardFilePath
+from emma.kb.kb_utils_refactor import KnowledgeBase
+
+
+# class for enriching entity with synonyms
+class SynonymEnricher:
+    def __init__(self):
+        """
+        Intialize
+        """
+        # Load synonym files
+        paths = StandardFilePath()
+        mesh_syn_file = os.path.join(paths.ontoemma_synonym_dir, 'mesh_synonyms.pickle')
+        dbpedia_syn_file = os.path.join(paths.ontoemma_synonym_dir, 'dbpedia_synonyms.pickle')
+
+        sys.stdout.write("Loading synonym files..\n")
+        self.mesh_synonyms = pickle.load(open(mesh_syn_file, 'rb'))
+        self.dbpedia_synonyms = pickle.load(open(dbpedia_syn_file, 'rb'))
+
+    def add_synonyms_to_entity(self, aliases: List):
```

Comment:
The name suggests that this method updates the entity object somehow but it doesn't. Consider renaming to, e.g., `get_synonyms_of_entity`.